### PR TITLE
feat(feedback): add setting form addition for feedback crash report notifications.

### PR DESCRIPTION
**Pull request by JoshFerge**
Frontend counterpart to https://github.com/getsentry/sentry/pull/65352, used in https://github.com/getsentry/sentry/pull/65561

### CODE REVIEW

File: static/app/data/forms/userFeedback.tsx [hunk 1]
```diff
@@ -18,6 +18,14 @@ const formGroups: JsonFormObject[] = [
         help: 'Show "powered by Sentry within the feedback dialog. We appreciate you helping get the word out about Sentry! <3',
         getData: data => ({options: data}),
       },
+      {
+        name: 'sentry:feedback_user_report_notifications',
+        type: 'boolean',
+
+        label: 'Enable Crash Report Notifications',
```
Comment (bruno-garcia) [inline static/app/data/forms/userFeedback.tsx:25]: Can we link to the docs section that explains what "crash report" is? I feel it's not that obvious for new Sentry users. Also perhaps "Modal" there somewhere helps folks realize what this is about

```suggestion
        label: 'Enable Crash Report Modal Notifications',
```

**Review by bruno-garcia — APPROVED** [review]

File: static/app/data/forms/userFeedback.tsx [hunk 2]
```diff
@@ -18,6 +19,23 @@ const formGroups: JsonFormObject[] = [
...
+        help: () =>
+          tct(
+            'Get notified on [crashReportModalDocsLink: Crash Report Modal and User Report API submissions]. Feedback widget notifications are not affected by this setting and are on by default.',
+            {
+              crashReportModalDocsLink: (
+                <a href="https://docs.sentry.io/platforms/javascript/user-feedback/#crash-report-modal" />
+              ),
+            }
+          ),
+        // help: 'Get notified on Crash Reports and User Report API submissions. Feedback widget notifications are not affected by this setting and are on by default.',
```
Comment (ryan953) [inline static/app/data/forms/userFeedback.tsx:36]:
```suggestion
```

File: static/app/data/forms/userFeedback.tsx [hunk 3]
```diff
@@ -18,6 +19,23 @@ const formGroups: JsonFormObject[] = [
         help: 'Show "powered by Sentry within the feedback dialog. We appreciate you helping get the word out about Sentry! <3',
         getData: data => ({options: data}),
       },
+      {
+        name: 'sentry:feedback_user_report_notifications',
+        type: 'boolean',
+
+        label: 'Enable Crash Report Notifications',
```
Comment (ryan953) [inline static/app/data/forms/userFeedback.tsx:26]:
```suggestion
        label: t('Enable Crash Report Notifications'),
```

Comment (ryan953) [reply static/app/data/forms/userFeedback.tsx:26]: should be translating the label & help text for "Show Sentry Branding" above too

**Review by ryan953 — APPROVED** [review]

**Pull request merged**